packages/web-api/src/methods.ts

```diff
@@ -2170,6 +2172,14 @@ export abstract class Methods extends EventEmitter<WebClientEvent> {
     messages: bindApiCall<SearchMessagesArguments, SearchMessagesResponse>(this, 'search.messages'),
   };
 
+  public readonly slacklists = {
```

Comment:
```suggestion
  public readonly slackLists = {
```

👾 issue: Uppercasings to match the upstreams here